third_party/terraform/resources/resource_google_service_account.go

```diff
@@ -114,19 +123,21 @@ func resourceGoogleServiceAccountDelete(d *schema.ResourceData, meta interface{}
 
 func resourceGoogleServiceAccountUpdate(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*Config)
-	if ok := d.HasChange("display_name"); ok {
+	if d.HasChange("display_name") || d.HasChange("description") {
 		sa, err := config.clientIAM.Projects.ServiceAccounts.Get(d.Id()).Do()
 		if err != nil {
 			return fmt.Errorf("Error retrieving service account %q: %s", d.Id(), err)
 		}
 		_, err = config.clientIAM.Projects.ServiceAccounts.Update(d.Id(),
 			&iam.ServiceAccount{
 				DisplayName: d.Get("display_name").(string),
+				Description: d.Get("description").(string),
 				Etag:        sa.Etag,
 			}).Do()
 		if err != nil {
 			return fmt.Errorf("Error updating service account %q: %s", d.Id(), err)
 		}
+		time.Sleep(time.Second)
```

Comment:
Seeing sleep's in a test makes me uncomfortable, sleep's in production code much more so. Is this only to make the test pass?

Reply:
Nope, the test passes about 70% of the time without it.  It's just that this api is only mostly synchronous.  This is more than enough time to get a consistent result, which is good for users and tests.  Costs very little, even though it's terrible practice.

Reply:
Oh dear. Can you add a comment inline otherwise without context I can see this being removed by a well meaning future developer.